README.md

```diff
@@ -37,6 +37,17 @@ The Amazon Genomics CLI is written in Go.
 To manage and install Go versions, we use [goenv](https://github.com/syndbg/goenv). Follow the installation
 instructions [here](https://github.com/syndbg/goenv/blob/master/INSTALL.md).
 
+Once goenv is installed, use it to install the version of Go required by the
+Amazon Genomics CLI build process, so that it will be available when the build
+process invokes goenv's `go` shim:
+
+```bash
+goenv install $(cat .go-version)
```

Comment:
If you're anywhere in the AGC directory it will install the correct version by looking up your path tree for the first `.go-version` file. Just `goenv install` is sufficient here.

Reply:
OK, I've changed to the shorter command.